.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,29 @@
+name: Bug Report
+description: Create a report to help reproduce and fix the issue
+title: "Bug: "
+labels: ["bug"]
+body:
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Reproduction
+      description: How did you manage to make the error happen?
+      placeholder: How did you do it?
+      value: "1. Run StrongDMM v2.0.0.alpha"
```

Comment:
Can you remove the version value from here and add an additional text field where user need to provide the version number? Better to put this field at the beginning.